### PR TITLE
Update did-io to pluggable Chai like architecture

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
- No did methods bundled with did-io by default
- Require did-io, pass in did methods you'll be using via `.use(method, driver)` calls.
- After that, common API, just call `.get(did)` etc.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # did-io ChangeLog
 
+## 0.9.0
```
Comment (davidlehn) [inline CHANGELOG.md]: Is 0.9.0 needed if 0.8.0 was never released?

File: README.md [hunk 2]
```diff
@@ -91,10 +91,20 @@ See the [Veres One Method spec](https://w3c-ccg.github.io/didm-veres-one/) for
...
+const veresOneDriver = v1.driver({ mode: 'test', httpsAgent, documentLoader });
+
+const keyDriver = require('did-key-driver').driver({});
+
 const dids = require('did-io');
 
-const v1 = dids.methods.veres({ mode: 'test' });
-// or use mode: 'dev' when testing against a local Veres One node
+dids.use('did:v1', veresOneDriver);
+dids.use('did:key', keyDriver);
```
Comment (davidlehn) [inline README.md]: Why the `did:` prefix vs just `v1` and `key`?

File: CHANGELOG.md (hunk 1 above)
Comment (dmitrizagidulin) [reply CHANGELOG.md]: Oh. Uh.. good point.

File: README.md (hunk 2 above)
Comment (dmitrizagidulin) [reply README.md]: I'm thinking - to make it super explicit that it's a did method? You're right that it's not required really.

Comment (dmitrizagidulin) [reply README.md]: Changed it to just use `v1` instead of `did:v1`.

**Review by dlongley — APPROVED** [review]
LGTM. One nit is to request that we use `async/await` in examples instead of Promise `.then` APIs.

**Comment by dlongley** [comment]
@dmitrizagidulin -- can you please handle the nits and get this merged?

File: README.md [hunk 3]
```diff
@@ -116,29 +116,29 @@ const didIo = require('did-io');
...
 // See did-veres-one repo for instructions on how to set up the httpsAgent etc
-const veresOneDriver = v1.driver({ mode: 'dev', httpsAgent, documentLoader });
+const veresDriver = v1.driver({ mode: 'dev', httpsAgent, documentLoader });
 
 // to use the did:v1 / Veres One method
 didIo.use('v1', veresOneDriver);
 
 // Now you can start using the API
-didIo.get({ did }).then(didDoc => { console.log(didDoc); }); 
+const didDoc = didIo.get({did});
```
Comment (dlongley) [inline README.md]: Missing `await`

Comment (dmitrizagidulin) [reply README.md]: ::facepalms:: Thanks. Fixed.

**Pull request merged**